docs/system/container-metrics.md

```diff
@@ -23,15 +23,20 @@ This metric is [opt-in][MetricOptIn].
 <!-- semconv metric.container.cpu.time(full) -->
 | Attribute  | Type | Description  | Examples  | [Requirement Level](https://opentelemetry.io/docs/specs/semconv/general/attribute-requirement-level/) | Stability |
 |---|---|---|---|---|---|
-| [`container.cpu.state`](/docs/attributes-registry/container.md) | string | The CPU state for this data point. A container SHOULD be characterized _either_ by data points with no `state` labels, _or only_ data points with `state` labels. | `user`; `kernel` | `Opt-In` | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| [`cpu.state`](/docs/attributes-registry/cpu.md) | string | The CPU state for this data point. A container SHOULD be characterized _either_ by data points with no `state` labels, _or only_ data points with `state` labels. | `user`; `system` | `Opt-In` | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
```

Comment:
I don't see this attribute deleted from the container registry.
the same for process and systen

Reply:
Those shouldn't be just deleted (as we don't support deletion anymore) but deprecated

Reply:
Yeap good point, thank's!

Reply:
I think this is fixed now. Closing this comment.